tests/vmi_multus_test.go

```diff
@@ -338,8 +344,15 @@ var _ = Describe("Multus Networking", func() {
 
 			By("checking virtual machine instance has three interfaces")
 			checkInterface(vmiOne, "eth0", "#")
-			checkInterface(vmiOne, "eth1", "#")
-			checkInterface(vmiOne, "eth2", "#")
+			checkInterface(vmiOne, "net1", "#")
+			checkInterface(vmiOne, "net2", "#")
+
+			By("mouting cloudinit iso")
+			mountCloudInit(vmiOne, "#")
+
+			By("checking cloudinit network-config contains interface")
+			checkNetworkConfig(vmiOne, "net1", "#")
+			checkNetworkConfig(vmiOne, "net2", "#")
```

Comment:
Can you also add a specific functional test case to validate behavior when ```networkData``` is set in spec?

Reply:
Will do.

Reply:
Done